keg_auth/cli.py

```diff
@@ -38,4 +41,31 @@ def _create_user(as_superuser, no_mail, **kwargs):
         create_user = click.argument(arg)(create_user)
     auth.command('create-user')(create_user)
 
+    @click.argument('username')
+    @click.option('--attempt-type', '--type')
+    def purge_attempts(username, attempt_type):
+        auth_manager = keg.current_app.auth_manager
+        try:
+            attempt_ent = auth_manager.entity_registry.attempt_cls
+        except RegistryError:
+            click.echo('No attempt class has been registered.')
+            return
+
+        user_ent = auth_manager.entity_registry.user_cls
+        try:
+            user = user_ent.get_by(username=username) or user_ent.get_by(email=username)
+        except InvalidRequestError:
+            user = None
+
+        assert user is not None, 'No user found with username "{}"'.format(username)
+
+        attempt_query = attempt_ent.query.filter_by(user_id=user.id)
```

Comment:
Once we have a user validated and in scope, I'd want to see this part of it moved to the Attempt model. I.e. I'd like to have this method available for use elsewhere in an application.